chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -138,7 +138,7 @@
             <job-data-map>
                 <entry>
                     <key>generateCsvFile</key>
-                    <value>true</value>
+                    <value>false</value>
```

Comment:
Is this for ease of debugging locally?

Reply:
Yes - generation of the CSV with all of the historical data take time.  Just generating the current statistics only takes a few seconds.

Reply:
Yes - generation of the CSV with all of the historical data take time. Just generating the current statistics only takes a few seconds.